memcached.c

```diff
@@ -11857,12 +11869,12 @@ static void process_mop_prepare_nread(conn *c, int cmd, char *key, size_t nkey,
         c->coll_field  = *field;
         conn_set_state(c, conn_nread);
     } else {
-        if (cmd == OPERATION_MOP_INSERT) {
+        if (cmd == OPERATION_MOP_INSERT || cmd == OPERATION_MOP_UPSERT) {
             if (settings.detail_enabled) {
                 stats_prefix_record_mop_insert(key, nkey, false);
             }
             STATS_CMD_NOKEY(c, mop_insert);
-        } else if (cmd == OPERATION_MOP_UPDATE) {
+        } else {
```

Comment:
주석을 추가해 두면 읽기 편할 것으로 생각
```suggestion
        } else /* OPERATION_MOP_UPDATE */ {
```

Reply:
수정되었습니다.